rmgpy/cantherm/molepro.py

```diff
@@ -42,7 +45,226 @@ class MoleProLog:
     
     def __init__(self, path):
         self.path = path
-            
+
+    def getNumberOfAtoms(self):
+        """
+        Return the number of atoms in the molecular configuration used in
+        the MolPro log file.
+        """
+
+        Natoms = 0
+        # Open Gaussian log file for parsing
+        f = open(self.path, 'r')
+        line = f.readline()
+        while line != '' and Natoms == 0:
+            # Automatically determine the number of atoms
+            if 'ATOMIC COORDINATES' in line and Natoms == 0:
+                for i in range(4): line = f.readline()
+                while 'Bond lengths' not in line:
+                    Natoms += 1
+                    line = f.readline()
+            line = f.readline()
+        # Close file when finished
+        f.close()
+        # Return the result
+        Natoms -= 1
+
+        return Natoms
+
+    def loadForceConstantMatrix(self):
```

Comment:
Hi Matt, I haven't made any changes to the molepro.py file yet. I am testing a couple of things for the F12 methods before making the changes. It will probably be changed in the next commit